src/Data/Managers/V2/GetCurrentUser/useGetCurrentUser.ts

```diff
@@ -0,0 +1,51 @@
+import { useQuery } from "@tanstack/react-query";
+import { PrimaryBaseUrlManager } from "@/UI";
+import { useFetchHelpers } from "../helpers";
+
+/**
+ * Custom hook for removing a user from the server.
+ *
+ * @returns {Mutation} - The mutation object provided by `useMutation` hook.
+ */
+export const useGetCurrentUser = () => {
```

Comment:
Why is it calle Get when the docstrings tell it's for removing a user from the server?